fedgov-cv-mapencoder/src/main/java/gov/usdot/cv/mapencoder/LaneDataAttribute.java

```diff
@@ -25,6 +25,14 @@ public class LaneDataAttribute {
     private int laneAngle;
     private SpeedLimitList speedLimits;
 
+    //enum choices
+    public static final int PATH_END_POINT_ANGLE = 1;
```

Comment:
Is there a reason we did not make this a regular java enumeration ? Something like 
```java
public enum LaneDataAttributeEnum {
    PATH_END_POINT_ANGLE(1),
    LANE_CROWN_POINT_CENTER(2),
    LANE_CROWN_POINT_RIGHT(3),
    LANE_CROWN_POINT_LEFT(4),
    LANE_ANGLE(5),
    SPEED_LIMITS(6);
    private int value;
    LaneDataAttributeEnum( int value) {
        this.value=value;
    }
    public int getValue() { return this.value;}
   
}
```